src/screens/rewards/RewardsSheet.tsx

```diff
@@ -41,10 +40,10 @@ export const RewardsSheet: React.FC = () => {
           scrollEnabled
         >
           <Box padding="20px">
-            {loading || rewardsData === null ? (
+            {isLoading || data === undefined || !data.rewards ? (
```

Comment:
should there be a different state (besides loading) if the user has no rewards?

Reply:
I think the default will be just displaying Zero

Reply:
But there will also be a version where an error occurred, I will ship it in a separate PR